app/views/sign_in/index.html.erb

```diff
@@ -6,9 +6,9 @@
     <h1 class="govuk-heading-l">Sign in to <%= I18n.t("service_name") %></h1>
 
     <% if AuthenticationService.dfe_signin? %>
-        <p class="govuk-body-m">Use DfE Sign-in to access your account. You need to be a registered administrator to login to these pages.</p>
+        <p class="govuk-body-m">Use DfE Sign-in to access your account.</p>
```

Comment:
![image](https://user-images.githubusercontent.com/470137/143063143-dc3eb5ca-8eeb-468b-90b0-be4c9dfab377.png)

I think we need to address the `admin` wording on the page

Reply:
Was going to make some changes here, but on second thought let's do that as part of https://trello.com/c/i4t2NzH7/606-publish-ttapi-merge-update-sign-in-page-copy-on-ttapi.